blog/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path, include
+
+from blog.views import IndexView, PostList, PostDetail
+
+urlpatterns = [
+    path("", IndexView.as_view(), name="index"),
+    path("posts/", PostList.as_view(), name="posts-list"),
+    path("posts/<int:pk>", PostDetail.as_view(), name="post-detail"),
```

Comment:
It's a good practice to add a trailing slash at the end of the URL pattern string for consistency and to avoid potential issues with URL resolution.